rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -0,0 +1,61 @@
+import random
+from http import HTTPStatus
+
+import pytest
+import requests
+
+from rotkehlchen.tests.utils.api import (
+    api_url_for,
+    assert_error_response,
+    assert_ok_async_response,
+    assert_proper_response_with_result,
+    wait_for_async_task_with_result,
+)
+
+
+@pytest.mark.parametrize('assets, error_message', [
```

Comment:
For such tests can you please make unit tests in some way?

These are going to slow down our tests A LOT.

So this single test, had a parametrization that turns it into 3 different tests. It's 3 different integration tests that setup an entire server, run the test and then tear the server down.

Same for all the rest of the tests.

We need to unit test whatever can be unit tested and stop testing everything in unit tests. Or else we will end up with our CI suite exceeding one hour.

Reply:
Damn true, ok, I'll do as unit or delete it. It was my first time with marshmallow + webargs (so rare right? but at previous jobs we had our own schema engine) so I wanted to be sure it was behaving as expected.

Reply:
you can test just the marshmallow filter is implemented properly somewhere in a unit test. But in the end you end up testing the underlying library and not your implementation.